03_launch_mgmt_cluster.sh

```diff
@@ -110,6 +110,10 @@ IRONIC_ENDPOINT=${IRONIC_URL}
 IRONIC_INSPECTOR_ENDPOINT=${IRONIC_INSPECTOR_URL}
 EOF
 
+  if [ -n "${DEPLOY_ISO_URL}" ]; then
+    echo "DEPLOY_ISO_URL=${DEPLOY_ISO_URL}" | sudo tee -a "${BMOPATH}/config/default/ironic.env"
+  fi
+
```

Comment:
why not to append this var in ? https://github.com/metal3-io/metal3-dev-env/blob/08950cc47e6460de941f21229808fade04c7468e/03_launch_mgmt_cluster.sh#L107-L110

Reply:
@fmuyassarov I made it conditional because unlike the kernel/ramdisk DEPLOY_ISO_URL isn't set by default - if we're OK with just appending an empty value then I can just add it to the list

Reply:
FYI it's needed for testing virtualmedia via https://github.com/metal3-io/ironic-agent-image/ - so it's not required by default but a user may want to export it when testing that path (and in future perhaps we may want to add fedora-coreos support to metal3-dev-env, but that's not my current aim, I'm just trying to make developer testing easier)

Reply:
Makes sense. I missed that we don't have anything by default for it. So, better to avoid empty value.